### DISCUSSION TITLE
Connection String in Flutter apps

### DISCUSSION THREAD

**Discussion by Othman-Sami** [supabase/supabase#20244] [Questions] [answered]
Hi every one
I have app made in flutter 
I use supabase project to have backend data

today Supabase told me in message that I should use new Supavisor in Conncetion String

How can I use it in Flutter ?

**Comment by GaryAustin1** [comment, marked as answer]
This is not relevant for the REST clients you would normally use for things like Flutter.   It is for things like Prisma where you use direct database pooling ports.

**Comment by jonasermert** [comment]
```
`dependencies:
  supabase: ^<latest_version>
`
```

```
import 'package:supabase/supabase.dart';

final supabase = SupabaseClient(
  'https://your-project-id.supabase.co',
  'your-supabase-api-key',
  connectTimeout: 5000, // Set your desired timeout
  readTimeout: 5000,    // Set your desired timeout
);

```
